doc/rados/operations/crush-map.rst

```diff
@@ -618,285 +618,280 @@ To remove a weight set:
-fine for most clusters, provided there are not many OSDs that have
-been marked out.
+The legacy CRUSH behavior used by Argonaut and older releases works fine for
+most clusters, provided that not many OSDs have been marked ``out``.
 
 bobtail (CRUSH_TUNABLES2)
 -------------------------
 
-The ``bobtail`` tunable profile fixes a few key misbehaviors:
+The ``bobtail`` tunable profile provides the following fixes:
 
- * For hierarchies with a small number of devices in the leaf buckets,
-   some PGs map to fewer than the desired number of replicas.  This
-   commonly happens for hierarchies with "host" nodes with a small
-   number (1-3) of OSDs nested beneath each one.
+ * For hierarchies with a small number of devices in the leaf buckets, some PGs
+   might map to fewer than the desired number of replicas.  This is known to
+   happen in the case of hierarchies with "host" nodes that have a small number
+   of OSDs (1 to 3) nested beneath each host.
 
- * For large clusters, some small percentages of PGs map to fewer than
-   the desired number of OSDs.  This is more prevalent when there are
-   multiple hierarchy layers in use (e.g., ``row``, ``rack``, ``host``, ``osd``).
+ * For large clusters, a small percentage of PGs might map to fewer than the
+   desired number of OSDs. This is known to happen when there are multiple
+   hierarchy layers in use (for example,, ``row``, ``rack``, ``host``,
+   ``osd``).
 
- * When some OSDs are marked out, the data tends to get redistributed
+ * When one or more OSDs are marked ``out``, the data tends to be redistributed
    to nearby OSDs instead of across the entire hierarchy.
 
-The new tunables are:
+The new tunables introduced by ``bobtail`` are as follows:
 
- * ``choose_local_tries``: Number of local retries.  Legacy value is
-   2, optimal value is 0.
+ * ``choose_local_tries``: Number of local retries. The legacy value is ``2``,
+   and the optimal value is ``0``.
 
- * ``choose_local_fallback_tries``: Legacy value is 5, optimal value
-   is 0.
+ * ``choose_local_fallback_tries``: The legacy value is ``5``, and the optimal
+   value is 0.
 
- * ``choose_total_tries``: Total number of attempts to choose an item.
-   Legacy value was 19, subsequent testing indicates that a value of
-   50 is more appropriate for typical clusters.  For extremely large
-   clusters, a larger value might be necessary.
+ * ``choose_total_tries``: Total number of attempts to choose an item.  The
+   legacy value is ``19``, but subsequent testing indicates that a value of
+   ``50`` is more appropriate for typical clusters. For extremely large
+   clusters, an even larger value might be necessary.
 
- * ``chooseleaf_descend_once``: Whether a recursive chooseleaf attempt
-   will retry, or only try once and allow the original placement to
-   retry.  Legacy default is 0, optimal value is 1.
+ * ``chooseleaf_descend_once``: Whether a recursive chooseleaf attempt will
```

Comment:
``chooseleaf``?

Reply:
I don't know. @neha-ojha?

Reply:
I meant to double-backtick on like 797, markup only.

Reply:
Accepted.